lib/extensions/attachable/action_view/helpers/form_builder.rb

```diff
@@ -2,9 +2,12 @@
 module Extensions::Attachable::ActionView::Helpers::FormBuilder
   # Method from ActsAsAttachable framework.
   # Hepler to support f.attachments in form
-  def attachments
+  #
+  # @param [Boolean] allow_delete Specify if attachments can be deleted.
+  def attachments(allow_delete = true)
```

Comment:
better to use keyword arguments for booleans.   `f.attachments(allow_delete: false)` is more readable than `f.attachments(false)`